invenio_files_rest/utils.py

```diff
@@ -21,6 +21,9 @@
 }
 """Mapping encoding to MIME types which are not in mimetypes.types_map."""
 
+FALLBACK_MIMETYPE = "application/octet-stream"
```

Comment:
does it make sense to expose it as config ? ping @slint

Reply:
Actually, this doesn't make sense to have as config, the fallback for "I've no idea what I'm serving" is `application/octet-stream` (i.e. "you get bytes").